plugin/input/journalctl/reader.go

```diff
@@ -59,6 +62,7 @@ func readLines(r io.Reader, config *journalReaderConfig) {
 	}
 }
 
+//nolint:deadcode,unused
```

Comment:
if it's dead code, let's remove it

Reply:
journalctl linux only, that's for golangci on mac/win